libcalico-go/lib/backend/k8s/resources/workloadendpoint_test.go

```diff
@@ -89,7 +89,8 @@ var _ = Describe("WorkloadEndpointClient", func() {
 					Value: wep,
 				}
 
-				_, err = wepClient.Create(context.Background(), kvp)
+				ctxCNI := resources.ContextWithPatchMode(context.Background(), resources.PatchModeCNI)
```

Comment:
Maybe we could default to `PatchModeCNI` if not specified for backward compatibility, and to avoid needing to modify all of these contexts? 

Not really a big deal, just a thought.

Reply:
I thought about this too, but @fasaxc and I thought that being explicit in all calls would be clearer what was happening.